src/views/Restoration/RestorationBusinessInformation.vue

```diff
@@ -202,6 +108,22 @@ export default class RestorationBusinessInformation extends Vue {
     })
   }
 
+  private isEmptyRecordsAddress () : boolean {
+    const recordsAddress = this.addresses.recordsOffice
+    if (!recordsAddress.mailingAddress ||
```

Comment:
If `recordsAddress` itself is null/undefined, this will throw an exception.

Ditto for `registeredAddress` below.

Reply:
Oh yes! Had the check there not sure why I removed it :(
Adding it back

Reply:
You could use [optional chaining](https://www.google.com/search?q=optional+chaining) to simplify the code. Up to you.

Reply:
Done